google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -109,25 +104,31 @@ public ProjectSelector(final boolean queryOnExpand) {
     projectSelectionListeners = new ArrayList<>();
 
     // synchronize selection between the treemodel and current text.
-    treeModel.addTreeModelListener(new TreeModelAdapter() {
-      @Override
-      public void treeStructureChanged(TreeModelEvent event) {
-        if (!Strings.isNullOrEmpty(getText())
-            && popup != null && !popup.isDisposed() && popupPanel != null
-            && event.getTreePath() != null
-            && event.getTreePath().getLastPathComponent() instanceof GoogleUserModelItem) {
-          GoogleUserModelItem userItem = (GoogleUserModelItem) event.getTreePath()
-              .getLastPathComponent();
-          for (int index = 0; index < userItem.getChildCount(); index++) {
-            DefaultMutableTreeNode loadedItem = (DefaultMutableTreeNode) userItem.getChildAt(index);
-            if (loadedItem instanceof ResourceProjectModelItem
-                && getText().equals(((ResourceProjectModelItem) loadedItem).getProjectId())) {
-              popupPanel.tree.setSelectionPath(new TreePath(loadedItem.getPath()));
+    treeModel.addTreeModelListener(
```

Comment:
doesn't look like it, but just to make sure I didn't miss anything: nothing was changed in this block other than formatting right?

Reply:
Whoops, yeah you're right. Sorry about that, auto formatter must have kicked in and I didn't notice it.